.vsts-ci.yml

```diff
@@ -0,0 +1,134 @@
+resources:
+- repo: self
+
+phases:
+- phase: linux_trusty_openssl
+  displayName: 'Linux (Trusty OpenSSL)'
+  queue:
+    name: 'Hosted Linux Preview'
+  steps:
+  - task: Docker@0
+    displayName: Build
+    inputs:
+      action: 'Run an image'
+      imageName: 'ethomson/libgit2-trusty-openssl:latest'
+      volumes: |
+       $(Build.SourcesDirectory):/src
+       $(Build.BinariesDirectory):/build
+      workDir: '/build'
+      containerCommand: '/src/ci/build.sh'
+      detached: false
+  - task: Docker@0
```

Comment:
Is it intended that these tasks both have the same identifier? I don't really know the semantics, just left me wondering

Reply:
It's not an identifier, it's the type of task to run - this is a Docker task, version 0.